pipelines/vars/install_base.yml

```diff
@@ -1,3 +1,7 @@
 forklift_name: "pipeline-{{ pipeline_type }}-{{ pipeline_version }}-{{ pipeline_os }}"
 forklift_server_name: "pipeline-{{ pipeline_type }}-server-{{ pipeline_version }}-{{ pipeline_os }}"
 forklift_proxy_name: "pipeline-{{ pipeline_type }}-proxy-{{ pipeline_version }}-{{ pipeline_os }}"
+forklift_smoker_name: "pipeline-{{ pipeline_type }}-smoker-{{ pipeline_version }}-{{ pipeline_os }}"
+smoker_box:
```

Comment:
this could be defined in `base.yml`, no? or do we need different smoker boxes for install and upgrade?

Reply:
I don't see a `base.yml`. Are you suggesting to introduce one?